R/get_sample_cn_segments.R

```diff
@@ -18,130 +19,99 @@
 #'
 #' @return A data frame of segments for a specific or multiple sample ID(s).
 #'
-#' @import dplyr readr RMariaDB DBI glue GAMBLR.helpers
+#' @import dplyr readr RMariaDB DBI
```

Comment:
I would revert this back to mention the helpers and glue :smile: This is important because then they are in the NAMESPACE and will be checked if available as dependency when the package is installed

Reply:
I'm not sure, but if all glue and GAMBLR.helpers are prefixed in this function, and if both glue and GAMBLR.helpers are in the `Imports` field of `DESCRIPTION`, there is no need for a mention in NAMESPACE.

Reply:
Yeah. This is the case of how `get_gambl_colours` (from GAMBLR.helpers) is used in GAMBLR.utils. It is only prexied in the function code and GAMBLR.helpers is in the Imports of DESCRIPTION.